### DISCUSSION TITLE
Template_name on filter objects

### DISCUSSION THREAD

**Discussion by gone** [carltongibson/django-filter#1547] [Ideas]
Now that forms accept a template_name property to set the template to render, I think filtersets should also accept that property, and pass along to the form they generate.

Happy to put a pr in if it makes sense

**Comment by gone** [comment]
The way to do this w/o updating the filterset would be: 

```
class FilterForm(forms.Form):
    template_name = "account/member_filter_form.jinja"

class FilterSet(django_filters.FilterSet):
   
    class Meta:
        form = FilterForm
```

**Reply by carltongibson** [reply]
Hi. Thanks for the suggestion @gone. 

Using the existing `form` option is the way to go here (if you're wanting override your project's defaults)

https://django-filter.readthedocs.io/en/stable/ref/filterset.html#form
